src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -249,8 +278,8 @@ object LeonardoJsonSupport extends SprayJsonSupport with DefaultJsonProtocol {
   implicit val ipFormat = StringValueClassFormat(IP, IP.unapply)
   implicit val firewallRuleNameFormat = StringValueClassFormat(FirewallRuleName, FirewallRuleName.unapply)
 
-  implicit val clusterFormat = jsonFormat13(Cluster.apply)
-  implicit val clusterRequestFormat = jsonFormat4(ClusterRequest)
+  implicit val clusterFormat = jsonFormat20(Cluster.apply)
```

Comment:
here's another reason to break this out into sub-objects -- the jsonFormat functions only go up to 22!

Reply:
We could always implement our own `RootJsonFormat` to get around this limitation -- but yeah I agree it would be good to box up the cluster config params into its own object. :)

Reply:
Yeah this was definitely making me uncomfortable...